configs/jobs/job_train/job_train_objdet/job_train_objdet_number.yaml

```diff
@@ -1,5 +1,7 @@
 # train object detection with numbers dataset
 defaults:
+  # lock data location
+  - ops/filelocks@ops.acquire_locks.config.filelock_dict.data_lock: op_data_readlock.yaml
```

Comment:
Is `filelocks` ok as the directory of this file? I assume that this configuration file is used by several job configurations.
I can remember that there is a configuration directory called `share`.

Reply:
You are right!